dotnet/src/Connectors/Connectors.AI.OpenAI/OpenAIRequestSettings.cs

```diff
@@ -98,6 +99,18 @@ public string ChatSystemPrompt
     [JsonPropertyOrder(9)]
     public IDictionary<int, int> TokenSelectionBiases { get; set; } = new Dictionary<int, int>();
 
+    /// <summary>
+    /// Indicates a specific function that OpenAI should use to respond to the chat request.
+    /// If you want the model to choose from multiple functions, leave this null and
+    /// instead set <see cref="OpenAIRequestSettings.Functions"/>.
+    /// </summary>
+    public OpenAIFunction? FunctionCall { get; set; } = null;
```

Comment:
Not sure if these need the Json decorations and to be included in the converter. I don't imagine these being defined in prompts/text files but maybe good to keep the option open.

Reply:
cc @markwallace-microsoft

Reply:
I think it's OK to leave this out for the initial PR and then add it later as needed